sklearn/model_selection/tests/test_search.py

```diff
@@ -1004,6 +1130,10 @@ def test_predict_proba_disabled():
     gs = GridSearchCV(clf, {}, cv=2).fit(X, y)
     assert_false(hasattr(gs, "predict_proba"))
 
+    # Multi-metric case
```

Comment:
I don't see how extending this test helps more than `compare_refit_methods_when_refit_with_acc`